### ISSUE TITLE
Add support for the Web Treebank style of traces

### ISSUE THREAD

**Issue by jkkummerfeld** [jkkummerfeld/1ec-graph-parser#1]
More recent LDC data has annotated traces like this:

```
(S
  (NP-1 (NNP Bob) )
  (VP (VBD tried)
    (S
      (NP-1 (-NONE- *PRO*) )
      (VP (TO to)
        (VP (VB wash)
          (NP (PRP himself) ))))))
```

The format converter should be updated to handle this format too.

**Comment by Fantabulous-J** [comment]
It seems that the format converter doesn't handle some cases in OntoNotes 5.0 as you mentioned above. Do you plan to update the converter?

**Comment by jkkummerfeld** [comment]
I don't have any immediate plans to implement this (too busy with other work). Depending on what you need to do, the code does work for some cases. For example, it can read in the format and (1) print out one-line or multi-line versions of parses, and/or (2) apply most of the edits, and/or (3) filter by sentence length.

The main things it can't do are:
- Read ontonotes and print to my split head format
- Read my split head format and print to ontonotes format
- Read PTB and print ontonotes format

**Comment by Fantabulous-J** [comment]
Yeah, I can successfully read the parse tree in PTB format and convert it to your split head format on most cases of Ontonotes 5.0. It only fails in some rare cases. Maybe I can do it manually. But overall, thanks for your contribution

**Comment by jkkummerfeld** [comment]
Probably the easiest way to do this would be to write something to preprocess the parses, moving the indexation to the child node where appropriate (and then similar code to reverse that when going the other way). That said, there may be more complex examples than the one I included above.
